<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="outputFileActionComboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="timerModeCheckBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Работать по таймеру?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pollingIntervalSpinBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Интервал опроса</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Статус: Ожидание</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xorKeyLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите значение 8 байт для выполнения операции</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите действие</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>173</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выберите исходный файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filePathLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseFileButton">
       <property name="text">
        <string>Обзор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Выберите путь для сохранения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="savePathLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="savePathButton">
       <property name="text">
        <string>Выбрать путь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="deleteFilesCheckBox">
       <property name="text">
        <string>Удалить исходный файл ?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="processFilesButton">
       <property name="text">
        <string>Обработать файлы</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
